src/syft/core/common/module.py

```diff
@@ -0,0 +1,84 @@
+# stdlib
+from collections import OrderedDict
+from typing import Any
+from typing import List
+from typing import Optional
+from typing import Union
+
+
+# circular imports when using the syft.lib.full_name_with_qualname version
```

Comment:
Really? Isn't `full_name_with_qualname` in `src/syft/lib` which doesn't much?

Reply:
Yeah I tried, so weird. I guess its the other way around and the sy.Module itself is causing the issue?
<img width="1074" alt="Screen Shot 2020-10-29 at 8 40 54 am" src="https://user-images.githubusercontent.com/2882739/97504372-76a7f280-19c2-11eb-870c-802f8673ef39.png">